src/main/java/uk/gov/justice/hmpps/prison/service/transformers/OffenderTransformer.java

```diff
@@ -63,9 +68,18 @@ public InmateDetail transform(final OffenderBooking latestBooking) {
                     .question(pd.getId().getType().getDescription())
                     .resultValue(pd.getCode().getDescription())
                     .build()).toList())
+            .legalStatus(latestBooking.getLegalStatus())
+            .imprisonmentStatus(latestBooking.getActiveImprisonmentStatus().map(ims -> ims.getImprisonmentStatus().getStatus()).orElse(null))
+            .imprisonmentStatusDescription(latestBooking.getActiveImprisonmentStatus().map(ims -> ims.getImprisonmentStatus().getDescription()).orElse(null))
+            .offenceHistory(offenceHistory)
+            .recall(RecallCalc.calculate(latestBooking.getBookingId(), latestBooking.getLegalStatus(), offenceHistory, sentenceTerms))
+            .receptionDate(latestBooking.getBookingBeginDate().toLocalDate())
+            .locationDescription(LocationProcessor.formatLocation(latestBooking.getLocation().getDescription()))
+            .latestLocationId(latestBooking.getLocation().getId())
             .build()
             .deriveStatus()
-            .splitStatusReason();
+            .splitStatusReason()
+            .updateReligion();
```

Comment:
Followed the existing pattern for derived data to get the religion - but this will also need moving out of InmateService soon.